### ISSUE TITLE
Update Server Deployment Documentation

### ISSUE THREAD

**Issue by GroundPound** [picoCTF/picoCTF-Platform-2#11]
At the least we could use some notes on dependencies, updating the configuration, and starting the web server.

**Comment by rvantonder** [comment]
1. For deploying this live, I modified `vagrant_setup.sh` to use the correct directory--as a first step, I suggest extracting `/home/vagrant` out to a bash variable that can be changed as desired.

2. Cookies don't work unless you change `api.app.session_cookie_domain` to point to "yoursite.com"` in  `api/api/config.py`

3. I find my own script to reload problems immensely useful, but I don't know if there's a better way to remove problems/submissions other than the mongo way. I put this script in `api/repopulate-problems.sh`:

```
 mongo pico --eval "db.problems.remove()"
 mongo pico --eval "db.submissions.remove()"

python3 api_manager.py -v problems load ~/problems/ graders/ ../problem_static/
python3 api_manager.py autogen build 10
devploy
```
Note that `~/problems/` needs to point to the right directory contain the problems, so this is not a robust example.

4. You need to 'enable' the scoreboard once everything is ready. The current way to do this is to run `python3 daemon_manager.py cache_stats -i 300` in  its own process in the `api/` directory.

That's it for now.

**Comment by GroundPound** [comment]
@rvantonder , were you planning on integrating these tidbits into the documentation? If not let me know and I'll assign someone else to take it from here.

**Comment by rvantonder** [comment]
These are the notes that I thought would be useful (in my experience)--you're welcome to have someone incorporate it into the documentation where applicable

**Comment by royragsdale** [comment]
The merged documentation in #39 is the best effort at addressing this.

Please see [GettingStarted.md](https://github.com/picoCTF/picoCTF-Platform-2/blob/4cd86daa3f42b1be3e613d53d2861dfe987894f6/GettingStarted.md)
